components/brave_rewards/resources/ui/components/modalShowAdsHistory/index.tsx

```diff
@@ -162,6 +163,13 @@ export default class ModalShowAdsHistory extends React.PureComponent<Props, Stat
             allItems={this.state.filterStatus}
             header={this.headers}
           />
+          {
+            rows ?
```

Comment:
Is this intention here to show the "no activity" message if `rows` is empty? It looks like it will show the message if `rows` is "truthy" (meaning not undefined or null in this case).

I might be misunderstanding, but I would think this is something like what we want:

```ts
!rows || rows.length === 0 ?
```

(i.e. if we don't have a `rows` array or if the array is empty, then show the "no activity" message)

Reply:
Fixed